src/Magnum/BulletIntegration/MotionState.h

```diff
@@ -43,7 +43,31 @@ namespace Magnum { namespace BulletIntegration {
 
 Encapsulates `btMotionState` as @ref SceneGraph feature.
 
-@todoc Usage...
+# Usage
+
+Common usage is to either create a `btRigidBody` to share transformation with
+a @ref SceneGraph::Object by passing the motion state in its constructor:
+
+@code
+    SceneGraph::Object<SceneGraph::MatrixTransformation3D> object;
+    btRigidBody rigidBody{mass, &(new MotionState{object}).btMotionState(), collisionShape};
+
+    // rigidBody will be affected when changing the transform of object and
+    // object will be affected when transformation of rigidBody is changed.
+@endcode
```

Comment:
Please don't indent stuff inside `@code` -- it will get indented also in the documentation, wasting precious horizontal space.